modules/dxp/apps/segments/segments-experiment-web/src/main/java/com/liferay/segments/experiment/web/internal/product/navigation/control/menu/SegmentsExperimentProductNavigationControlMenuEntry.java

```diff
@@ -193,23 +192,12 @@ public boolean includeIcon(
 
 	public boolean isPanelStateOpen(HttpServletRequest httpServletRequest) {
 		String segmentsExperimentPanelState = SessionClicks.get(
-			httpServletRequest,
-			"com.liferay.segments.experiment.web_panelState", "closed");
+			httpServletRequest, _SESSION_CLICKS_KEY, "closed");
 
 		if (Objects.equals(segmentsExperimentPanelState, "open")) {
 			return true;
 		}
 
-		HttpServletRequest originalHttpServletRequest =
```

Comment:
Why do we have to remove this piece of code?

Reply:
I think this cannot be removed. It purpose is to open the panel when clicking on a notification. See https://github.com/liferay/liferay-portal/blob/f612f53a220b02c9ea534f4272947e7bb2b93df1/modules/dxp/apps/segments/segments-experiment-web/src/main/java/com/liferay/segments/experiment/web/internal/notifications/SegmentsExperimentUserNotificationHandler.java#L162-L163.